images/dockerregistry/Dockerfile

```diff
@@ -6,16 +6,20 @@
 #
 FROM openshift/origin-base
 
-COPY config.yml $REGISTRY_CONFIGURATION_PATH
-COPY bin/dockerregistry /dockerregistry
+RUN INSTALL_PKGS="origin-dockerregistry" && \
+    yum install -y ${INSTALL_PKGS} && \
+    rpm -V ${INSTALL_PKGS} && \
+    yum clean all
 
-LABEL io.k8s.display-name="OpenShift Origin Image Registry" \
-      io.k8s.description="This is a component of OpenShift Origin and exposes a Docker registry that is integrated with the cluster for authentication and management."
+COPY config.yml ${REGISTRY_CONFIGURATION_PATH}
+
+LABEL io.openshift.tags="openshift,docker,registry" \
+      io.k8s.display-name="OpenShift Container Platform Image Registry" \
+      io.k8s.description="This is a component ofOpenShift Container Platform and exposes a Docker registry that is integrated with the cluster for authentication and management."
 
-# The registry doesn't require a root user.
 USER 1001
 EXPOSE 5000
 VOLUME /registry
 ENV REGISTRY_CONFIGURATION_PATH=/config.yml
 
-CMD DOCKER_REGISTRY_URL=${DOCKER_REGISTRY_SERVICE_HOST}:${DOCKER_REGISTRY_SERVICE_PORT} /dockerregistry ${REGISTRY_CONFIGURATION_PATH}
+CMD DOCKER_REGISTRY_URL=${DOCKER_REGISTRY_SERVICE_HOST}:${DOCKER_REGISTRY_SERVICE_PORT} /usr/bin/dockerregistry ${REGISTRY_CONFIGURATION_PATH}
```

Comment:
So if I move this as an ENV, `${DOCKER_REGISTRY_SERVICE_HOST}:${DOCKER_REGISTRY_SERVICE_PORT}` resolves to `:` whereas in `CMD` it resolves to an empty string. I am not sure why `CMD` does that - I have opened https://github.com/openshift/origin/pull/14070 where I am looking into getting rid of the ENV altogether since it's only used for testing.